src/tests/detect-tls-version.c

```diff
@@ -236,15 +252,106 @@ static int DetectTlsVersionTestDetect02(void)
     PASS;
 }
 
+/*
+ * Test that a TLS version 1.0 packet will match on a !1.1 version signature
+ */
+static int DetectTlsVersionTestDetect03(void)
```

Comment:
we're actively trying to move away from these types of unittests in favor of suricata-verify tests. The unittests that mimic traffic and setting up all the layers (packet, stream, flow, etc) are generally taking too many shortcuts that hurt us later.